app/phone/src/main/res/layout/activity_player.xml

```diff
@@ -16,19 +16,32 @@
 
     <LinearLayout
         android:id="@+id/progress_scrubber_layout"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginTop="64dp"
+        android:padding="10dp"
         android:background="@drawable/overlay_background"
         android:clickable="false"
         android:gravity="center"
         android:orientation="vertical"
         android:visibility="gone"
         tools:visibility="visible">
 
+        <ImageView
+            android:id="@+id/image_preview_gesture"
+            android:layout_width="272dp"
+            android:layout_height="153dp"
+            android:layout_gravity="center"
+            android:layout_marginBottom="10dp"
+            android:background="@android:color/transparent"
```

Comment:
this causes the preview to not show. thanks for your work! 👍🏼

Reply:
Okay, now it should work.